test/test_contrib.py

```diff
@@ -239,3 +240,59 @@ def test_gradcheck(self):
         input = utils.tensor_to_gradcheck_var(input)  # to var
         assert gradcheck(kornia.contrib.spatial_soft_argmax2d,
                          (input), raise_exception=True)
+
+
+class TestDSNT:
+    GAUSSIAN = torch.tensor([
+        [0.002969, 0.013306, 0.021938, 0.013306, 0.002969],
+        [0.013306, 0.059634, 0.098320, 0.059634, 0.013306],
+        [0.021938, 0.098320, 0.162103, 0.098320, 0.021938],
+        [0.013306, 0.059634, 0.098320, 0.059634, 0.013306],
+        [0.002969, 0.013306, 0.021938, 0.013306, 0.002969],
+    ])
+
+    def test_render_gaussian_2d(self):
+        expected = self.GAUSSIAN
+        actual = dsnt.render_gaussian_2d(torch.tensor([2.0, 2.0]),
+                                         torch.tensor([1.0, 1.0]),
+                                         (5, 5),
+                                         normalized_coordinates=False)
+        assert_allclose(actual, expected, rtol=0, atol=1e-5)
+
+    def test_render_gaussian_2d_normalized_coordinates(self):
+        expected = self.GAUSSIAN
+        actual = dsnt.render_gaussian_2d(torch.tensor([0.0, 0.0]),
+                                         torch.tensor([0.25, 0.25]),
+                                         (5, 5),
+                                         normalized_coordinates=True)
+        assert_allclose(actual, expected, rtol=0, atol=1e-5)
+
+    def test_js_div_loss_2d(self):
+        examples = [
```

Comment:
I'd use for this `@pytest.mark.parametrize` to generate separate test
Check: https://github.com/arraiyopensource/kornia/blob/master/test/geometry/test_conversions.py#L23